src/NodeCompiler/CompileNodeToValue.php

```diff
@@ -98,14 +98,27 @@ private function compileClassConstFetch(Node\Expr\ClassConstFetch $node, Compile
         $className = $node->class->toString();
 
         if ($nodeName === 'class') {
```

Comment:
Can we isolate this entire block into a `private` method? I'd like to avoid the added CCLOC of `compileClassConstFetch`

Reply:
Done